### ISSUE TITLE
"string.std_ci" doesn't work 

### ISSUE THREAD

**Issue by Likojupa** [strengejacke/sjPlot#578]
Hi everyone, 

I try to modify the name of the column for std_ci, using "string.std_ci" but it doesn't work. 

Here is the code without specifying the name of the column:

tab_model(RR, show.intercept = FALSE, show.est = F, show.std = TRUE, show.stat = T, show.r2 = F, show.obs = F, 
          string.std = "ß", string.p = "p", string.stat = "t", 
          p.threshold = c(0.05, 0.01, 0.001), p.style = c("both"), digits.p = 2, emph.p = F, 
          ci.hyphen = "&nbsp  &nbsp;")

![Capture d’écran 2019-12-19 à 12 02 19](https://user-images.githubusercontent.com/21331188/71168667-8cec0a80-2257-11ea-8b0b-401875fa0595.png)

Here is the code when I specify the name of the column:

tab_model(RR, show.intercept = FALSE, show.est = F, show.std = TRUE, show.stat = T, show.r2 = F, show.obs = F, 
          string.std = "ß", string.p = "p", string.stat = "t", 
          p.threshold = c(0.05, 0.01, 0.001), p.style = c("both"), digits.p = 2, emph.p = F, 
          ci.hyphen = "&nbsp  &nbsp;", string.std_ci = "YYY")

![Capture d’écran 2019-12-19 à 12 02 19](https://user-images.githubusercontent.com/21331188/71168772-bd33a900-2257-11ea-9953-3bb8309d5b15.png)

Here is the code when I specify the name of "string.ci":

tab_model(RR, show.intercept = FALSE, show.est = F, show.std = TRUE, show.stat = T, show.r2 = F, show.obs = F, 
          string.std = "ß", string.p = "p", string.stat = "t", 
          p.threshold = c(0.05, 0.01, 0.001), p.style = c("both"), digits.p = 2, emph.p = F, 
          ci.hyphen = "&nbsp  &nbsp;", string.std_ci = "YYY", string.ci = "BBB")

![Capture d’écran 2019-12-19 à 12 05 18](https://user-images.githubusercontent.com/21331188/71168818-d9cfe100-2257-11ea-820d-caa94a2b6fe2.png)

It works, at least it modifies the end of the column name. 

In short, what I would like to do just to remove of the word "standardized". Any idea how I could do that? 

Thanks :)

**Comment by strengejacke** [comment]
Do you have the latest package version?

**Comment by Likojupa** [comment]
Yes, 2.0.1. 

My R is and my 3.6.1, and my Rstudio is 1.1.419

**Comment by strengejacke** [comment]
The latest version should be 2.8.1, and that issue should have been fixed in version 2.8.0. For me, it works fine.

**Comment by Likojupa** [comment]
Oh yes, I updated Jtools instead of sjPlot. It works perfectly now! 

Thank you very much for your help :)

**Comment by strengejacke** [comment]
Glad that it works!
